src/crystal/system/win32/random.cr

```diff
@@ -0,0 +1,12 @@
+module Crystal::System::Random
+  def self.random_bytes(buf : Bytes) : Nil
+    buf.size.times do |i|
+      buf[i] = next_u
+    end
+  end
+
+  def self.next_u : UInt8
+    # Chosen by fair dice roll, guaranteed to be random.
```

Comment:
Let's not forget that :smile:

Reply:
Haha yeah, was nearly tempted to merge it after @ysbaddaden reviewed it positive.

Reply:
Honestly that would be fine. Windows support is in flux. You can barely do anything with the corelib, neither IO, GC and I suspect the event loop.

This is a preliminary attempt. Fixes such as a proper random and time support could come in subsequent pull requests. I'll happily jump on implementing Random::Secure for example.

Reply:
Sure, but if that was the case i'd still merge something that compiled but raised instead of returning `4`.